packages/medusa/src/api/routes/store/shipping-options/list-shipping-options.js

```diff
@@ -40,7 +40,12 @@ export default async (req, res) => {
       relations: ["region", "items", "items.variant", "items.variant.product"],
     })
 
-    const options = await shippingProfileService.fetchCartOptions(cart)
+    let options
+    if (cart.type === "swap" || cart.type === "claim") {
+      options = await shippingProfileService.fetchRMAOptions(cart)
+    } else {
+      options = await shippingProfileService.fetchCartOptions(cart)
```

Comment:
Similar to how we approach adding a shipping method, would it be an idea to move this to the service layer?